metadata/Cadastre/plss_quarter_quarter_sections.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah PLSS Quarter Quarter Sections GCDB
+
+## ID
+
+4b63af59-da17-40dd-bf40-2996c489fc4d
+
+## Brief Summary
+
+Polygon dataset of the smallest unit of the Public Land Survey System (PLSS), where each unit represents 40 acres of land.
+
+## Summary
+
+This dataset represents the smallest unit of the Public Land Survey System (PLSS). Each polygon represents approximately 40 acres of land and is labelled by its township and range. We used Geographic Coordinate Database ([GCDB](https://www.sciencebase.gov/catalog/item/4f4e4a7fe4b07f02db648dce)) base attributes to create the second division class (quarter quarter sections).
```

Comment:
**question**: should we replace `we` with `UGRC` in case these files are read without the context of who we might be?

Reply:
Sounds like a good idea to me. I'll make the change!